Dockerfile

```diff
@@ -1,13 +1,16 @@
 # ---------------
 # Install Dependencies
 # ---------------
-FROM node:16-stretch-slim as builder
+FROM node:16-buster-slim as builder
 
 ARG PLATFORM
 
 ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini-static-${PLATFORM} /tini
 RUN chmod +x /tini
```

Comment:
can be removed if you install it in runner stage

Reply:
Updated